.babelrc

```diff
@@ -1,4 +1,7 @@
 {
   "presets": ["@babel/env", "@babel/typescript"],
-  "plugins": ["@babel/proposal-optional-chaining"]
+  "plugins": [
+    "@babel/proposal-optional-chaining",
+    "@babel/plugin-proposal-class-properties"
```

Comment:
Vil man alltid være nødt til å installere ekstra babel plugins hvis man bruker nyere features som optional-chaining?

Reply:
Ja, babel har sluttet med presets som støtte stage 3 stuff. https://babeljs.io/blog/2018/08/27/7.0.0#tc39-proposals-https-githubcom-tc39-proposals-support

Reply:
Hmm, okei. Jeg prøvde å bruke importere `styled-components/macro` for å få med klassenavn i styled komponentene jeg lagde, men fikk masse feil. Fikk ikke helt pinpointa det, men kanskje det er fordi jeg måtte ha med en babel plugin da 😁